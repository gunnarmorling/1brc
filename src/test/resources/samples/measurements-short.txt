a;1.0
b;1.0
b;2.0
