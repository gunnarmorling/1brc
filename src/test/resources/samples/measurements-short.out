{a=1.0/1.0/1.0, b=1.0/1.5/2.0}
